pkg/shell/index.html

```diff
@@ -409,15 +409,10 @@ <h4 class="pf-c-alert__title credential-alert"></h4>
         <div id="content" role="main" class="area-ct-content" tabindex="-1">
           <!-- This is where the iframes appear -->
           <div class="curtains-ct blank-slate-pf" role="alertdialog" aria-labelledby="machine-error-heading" aria-describedby="machine-error-desc" hidden>
-            <div class="blank-slate-pf-icon">
-              <div class="spinner spinner-lg" hidden></div>
-              <i class="fa fa-exclamation-circle"></i>
-            </div>
             <h1 id="machine-error-heading"></h1>
             <p id="machine-error-desc"></p>
             <div class="blank-slate-pf-main-action">
-              <button id="machine-reconnect" class="pf-c-button pf-m-primary btn-lg" translate="yes">Reconnect</button>
-              <button id="machine-troubleshoot" class="pf-c-button pf-m-primary btn-lg" translate="yes">Troubleshoot</button>
+              <button id="machine-troubleshoot" class="pf-c-button pf-m-primary btn-lg" translate="yes">Login</button>
```

Comment:
Since it's an action/verb, should be **Log in**. Login is only used for the adjective
https://www.patternfly.org/v4/design-guidelines/content/terminology

Reply:
Fixed.